Task/Taskfile.yml

```diff
@@ -1,10 +1,12 @@
+---
 # https://taskfile.dev
 
-version: '3'
+version: "3"
 
 tasks:
   clean:
     cmds:
       - find . -type f -name '.DS_Store' -delete
       - find . -type f -name '.Thumbs.db' -delete
       - find . -type f -name 'super-linter.log' -delete
+      - find . -type d -name '.ruff_cache' -exec rm -r "{}" \;
```

Comment:
Added a cleanup task for .ruff_cache. Another option is to run ruff with --no-cache.